Plugins/NativeEngine/Source/VertexBuffer.cpp

```diff
@@ -60,7 +60,7 @@ namespace Babylon
         m_disposed = true;
     }
 
-    void VertexBuffer::Update(Napi::Env env, gsl::span<uint8_t> bytes)
+    void VertexBuffer::Update(Napi::Env env, gsl::span<uint8_t> bytes, size_t byteOffset)
```

Comment:
Do we need to handle `byteOffset` for the dynamic path?

Reply:
It's not part of this PR: only update in the context of Thin Instances.

Reply:
Added an exception when using byte offset with GPU VB.